public/js/components/FormFields/__snapshots__/FormFieldRadioButtons.spec.js.snap

```diff
@@ -0,0 +1,37 @@
+exports[`test Should render 1`] = `
+<div>
+  <label
+    className="form__label"
+    htmlFor="test">
+    test
+  </label>
+  <div
+    className="form__group form__group--radio">
+    <input
+      checked={false}
+      className="form__radio-btn"
+      name="test"
+      onChange={undefined}
+      type="radio"
+      value="One" />
+    <span
+      className="form__label form__label--radio">
+      One
+    </span>
+  </div>
+  <div
+    className="form__group form__group--radio">
+    <input
+      checked={false}
+      className="form__radio-btn"
+      name="test"
+      onChange={undefined}
```

Comment:
Do we want to pass an update function so we can verify it's being used in correct place?